nodejs8/package.json

```
@@ -71,7 +71,7 @@
     "url-pattern": "1.0.3",
     "uuid": "3.3.2",
     "validator": "10.8.0",
-    "vcap_services": "0.5.1",
+    "vcap_services": "^0.6.0",
```

We use explicit versions.
```suggestion
    "vcap_services": "0.6.0",
```